charts/stable/rtorrent-rutorrent/values.yaml

```diff
@@ -50,20 +50,45 @@ service:
     ports:
       rtinc:
         enabled: true
-        port: 50000
+        port: 50000      
+  phphealth:
```

Comment:
Service is not needed for probes at all.
If users don’t have to reach this port, remove.
Otherwise add it to questions